src/view/loader.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import Background from '../component/background';
+import { CopyText } from '../component/text';
+import { FormStretcher } from '../component/form';
+import Icon from '../component/icon';
+import { color } from '../component/style';
+
+const styles = StyleSheet.create({
+  copyTxt: {
+    flexDirection: 'row',
+    fontSize: 53,
+  },
+  boltBackground: {
+    height: 46,
+    width: 25,
+    marginLeft: 26,
+    backgroundColor: '#57038D',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  bolt: {
+    height: 37,
+    width: 19,
+  },
+});
+
+const LoaderView = () => (
+  <Background color={color.blackDark}>
+    <FormStretcher>
+      <CopyText style={styles.copyTxt}>
+        The fastest way to transfer Bitcoin.<Bolt />
+      </CopyText>
```

Comment:
We should add some padding left and right of the text. This is probably best achieved with an extra container of a fixed width that is centered by the `MainContent` parent component.

Reply:
Hopefully what i went with is what you meant by that, i made a fixed width container. lmk if not!